patterns/2-structured/discover-your-innersource.md

```diff
@@ -83,4 +67,23 @@ TBD
 
 ## Acknowledgements
 
-* Contributions from Russ Rutledge, Ofer Hermoni and Robert Hanmer
+* Russ Rutledge
+* Ofer Hermoni
+* Robert Hanmer
+
+## Alias
+
+* Not looking for stuff internally
+* Don't bother looking
+* Find it Inside
+
+## Discussion on
+
+* Comments may appear in the timeline but not with the file once it is edited (github)?
+* https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/60
+* If only X knew what X knew; the internal search engine is bad. No one can find anything because it is difficult to add things to the search index.
+* We have similar challenges getting data sources pulled into the search engine. PayPal is building a project called Agora to do this and they are planning to open source it.
```

Comment:
@Danese is Agora a project at PayPal?

It looks like maybe yes? <https://www.bis.org/about/bisih/topics/fmis/agora.htm>